site/blog/airship-shanghai.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Airship at the Open Infrastructure Summit Shanghai in November
+author: chris-hoge
+date: 2019-08-07T09:00:00.000Z
+category: Summit
+---
+You can meet the Airship community at the upcoming [Open Infrastructure
+Summit](https://www.openstack.org/summit/shanghai-2019/) in Shanghai, November
+4-6. The Airship community will join 30 other open source projects attending the
+Summit to collaborate in the open. If you want to be part of it too [register
+now](http://openstack.org/summit/shanghai-2019) before prices increase on August
+14 at 11:59pm PT.
+
+For 2019, the Summit has expanded and rebranded from the OpenStack Summit to the
+Open Infrastructure Summit, reflecting the large number of open source tools
+used to automate modern infrastructure and the challenges of integrating them.
+The agenda for the event is now live with a full track around edge computing and
+lifecycle management featuring a number of Airship talks.
```

Comment:
There isn't a whole track on lifecycle management. Perhaps rephrase to say "The agenda for the event is now live with a full track around edge computing including talks around lifecycle management featuring a number of Airship talks.